R/zzz.R

```diff
@@ -58,7 +58,7 @@ maybe_register_S3_methods <- function() {
 
     for (i in seq_along(req_pkgs_names)) {
       class <- req_pkgs_classes[[i]]
-      s3_register("tune::required_pkgs", class)
+      s3_register("generics::required_pkgs", class)
     }
   }
```

Comment:
Same comment as before, we probably don't need to lazy register them now. We shouldn't need the if statement requiring a minimum tune version either. The whole chunk can probably be removed

Reply:
Thanks for this explanation; I am clearer now on how this registering is working.